Evaluator/TA/ai_evaluator/ai.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, tentacles_setup_config):
         self.indicator = None
         self.source = None
         self.period = None
-        self.min_confidence_threshold = 0
+        self.min_confidence_threshold = 100
```

Comment:
This is the default value?

Reply:
yes, we don't want to round to 1 below 100% confidence by default

Reply:
I'm not sure I understand. I haven't seen any 100% confidence value sent by gpt